### PR TITLE
Fix: Codebase font fixed, and other fixes for #2138 PR

### PR DESCRIPTION
**Pull request by zeul22**
Problem/Use Case:
Tailwind CSS overwrites the default style provided by Docusaurus, affecting the markdown content's appearance.

Technical Implementation:

Changes Made:

Reverted to traditional CSS.
Removed Tailwind CSS.
Adjusted Docs structure.

Alternatives Considered:

Modifying Markdown Files:
Wrap headers in div components within Markdown files and apply custom styling using src/css/markdown.css.
Dismissed due to inconvenience and the need to style specific words/statements individually (e.g., links, warnings).

Tailwind CSS Integration: 
Attempted implementation but was unsuccessful.

Additional Context:
Docusaurus has indicated plans to integrate TailwindCSS themes in v3.0 ([Ref](https://docusaurus.io/blog/2022/08/01/announcing-docusaurus-2.0#whats-next))

### CODE REVIEW

File: docs/src/css/custom.css [hunk 1]
```diff
@@ -5,6 +5,10 @@
  */
 
 /* You can override the default Infima variables here. */
+/* @tailwind base;
+@tailwind components;
+@tailwind utilities; */
```
Comment (neubig) [inline docs/src/css/custom.css:10]:
```suggestion
```

Ditto

**Review by neubig — COMMENTED** [review]
Could you please paste some screenshots onto this issue to demonstrate what the new pages look like, just to confirm?

**Comment by zeul22** [comment]
Sure! Here are the screenshots.

image
image
image
image

**Review by neubig — APPROVED** [review]
Thanks, looks great!

**Pull request merged**